src/main/java/com/ostapiuk/core/providers/DataObjectsProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.ostapiuk.core.providers;
+
+import com.ostapiuk.business.model.UserEntity;
+import com.ostapiuk.core.utils.JsonParser;
+import org.testng.annotations.DataProvider;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class DataObjectsProvider {
+
+    public static final List<UserEntity> USERS = Objects.requireNonNull(JsonParser.getJsonEntity()).getUsers();
+
+    @DataProvider(name = "providePortalUsers")
+    public Iterator<Object[]> provideUsers() {
+        return Stream.of(DataObjectsProvider.getUsers()).iterator();
+    }
+
+    public static Object[][] getUsers() {
+        int usersAmount = USERS.size();
+        Object[][] users = new Object[usersAmount][];
+        for (int i = 0; i < usersAmount; i++) {
```

Comment:
There are a lot of libs that can transform an object or file to Json.
Please read about Gson tool